### PR TITLE
10324 update 686 url

### PR DESCRIPTION
**Pull request by lbai001**

### CODE REVIEW

**Comment by lbai001** [comment]
@jbalboni dont know if there's any where else the url needs to be changed, but currently not showing up on `~ / disability-benefits / apply / dependents / form-686c-dependents /`

**Comment by wyattwalter** [comment]
Looks like there a duplicate `/disability-benefits` in one of the paths.

File: content/pages/disability-benefits/apply/dependents/form-686c-dependents.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ layout: page-react.html
     <ul class="row va-nav-breadcrumbs-list columns" role="menubar" aria-label="Primary">
       <li><a href="/">Home</a></li>
       <li><a href="/disability-benefits/"> Disability Benefits </a></li>
-      <li><a href="/disability-benefits/686/dependent-status"> Declaration of Dependents </a></li>
+      <li><a href="/disability-benefits/disability-benefits/apply/dependents/form-686c-dependents/"> Declaration of Dependents </a></li>
```
Comment (jbalboni) [inline content/pages/disability-benefits/apply/dependents/form-686c-dependents.md]: Duplicate `disability-benefits` here.

**Review by jbalboni — APPROVED** [review]
One issue with breadcrumb path, looks good otherwise.

**Pull request merged**